molecule/test14/converge.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: Converge
+  hosts: all
+  tasks:
+    - name: "Include ansible-role-openhpc"
+      include_role:
+        name: "{{ lookup('env', 'MOLECULE_PROJECT_DIRECTORY') | basename }}"
+      vars:
+        openhpc_enable:
+          control: "{{ inventory_hostname in groups['testohpc_login'] }}"
+          batch: "{{ inventory_hostname in groups['testohpc_compute'] }}"
+          runtime: true
+        openhpc_slurm_control_host: "{{ groups['testohpc_login'] | first }}"
+        openhpc_slurm_partitions:
+          - name: "compute"
+            extra_nodes:
+              # Need to specify IPs as even with State=DOWN, if slurmctld can't lookup a host it just excludes it from the config entirely
+              # Can't add to /etc/hosts via ansible due to Docker limitations on modifying /etc/hosts
+              - NodeName: fake-x,fake-y
+                NodeAddr: 0.42.42.0,0.42.42.1
```

Comment:
Should we use internal IPs here? like 10.42.42.0?

Reply:
Does this also affect cloud nodes that do not exist?

Reply:
@JohnGarbutt does 9b04782 clarify why I'm using invalid IPs rather than internal ones?

It doesn't affect cloud nodes which don't exist as they will have been listed in the config as State=CLOUD, not State=DOWN. The former specifically means `slurmctld` knows it doesn't know how to contact them until they're "resumed".